### PR TITLE
TASK: Cleanup thumbnail service code

### PR DESCRIPTION
**Pull request by daniellienert**
Fixes some code smells and does some code cleanup.

### CODE REVIEW

File: Neos.Media/Classes/Domain/Service/ThumbnailService.php [hunk 1]
```diff
@@ -78,31 +75,17 @@ class ThumbnailService
     protected $thumbnailCache = [];
 
     /**
+     * @Flow\Inject
```
Comment (kdambekalns) [inline Neos.Media/Classes/Domain/Service/ThumbnailService.php]: `private` and property injection do not work together, same below.

File: Neos.Media/Classes/Domain/Service/ThumbnailService.php [hunk 2]
```diff
@@ -78,31 +75,17 @@ class ThumbnailService
...
      * @var LoggerInterface
      */
     private $logger;
```
Comment (kdambekalns) [inline Neos.Media/Classes/Domain/Service/ThumbnailService.php]: Either this
```suggestion
    protected $logger;
```
or keep the method injection…

File: Neos.Media/Classes/Domain/Service/ThumbnailService.php [hunk 3]
```diff
@@ -78,31 +75,17 @@ class ThumbnailService
...
 
     /**
+     * @Flow\Inject
      * @var ThrowableStorageInterface
      */
     private $throwableStorage;
```
Comment (kdambekalns) [inline Neos.Media/Classes/Domain/Service/ThumbnailService.php]: Either this
```suggestion
    protected $throwableStorage;
```
or keep the method injection…

**Comment by daniellienert** [comment]
Thanks @kdambekalns and @markusguenther. Tested retargeting onto 5.1, which is possible as it was branched out before 5.1 was branched.

**Comment by davidspiola** [comment]
oh... did't check the base branche and merged it into master, sorry :-(. Does it make sense to merge into 5.1 as well?

**Comment by kitsunet** [comment]
Just creates extra hassle now, I woudl say leave as is

**Pull request merged**